submissions/ayokofemi/Database 1/init.sql

```diff
@@ -0,0 +1,44 @@
+--for the foodTable--
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(20) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the vendorFood--
+CREATE TABLE vendor_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(30) NOT NULL,
+amount FLOAT(4,2) NOT NULL,
+PRIMARY KEY (id)
+);
```

Comment:
FOREIGN KEY is missing to Reference the admin_id for the admin table and food_id for the food table